### PR TITLE
Clarifies distinctions between getter and setter methods

### PR DESCRIPTION
**Pull request by lady3bean**
Fixes gh-655

### CODE REVIEW

**Comment by gnarf** [comment]
LGTM - cc @jquery/content for a second reviewer please

File: page/using-jquery-core/working-with-selections.md [hunk 1]
```diff
@@ -5,16 +5,17 @@
...
 ```
-// The .html() method used as a setter:
+// The .html() method sets all the h1 elements' html to be "hello world":
 $( "h1" ).html( "hello world" );
 ```
 
 ```
-// The .html() method used as a getter:
-$( "h1" ).html();
+// The .html() method returns the html from the first h1 element:
```
Comment (AurelioDeRosa) [inline page/using-jquery-core/working-with-selections.md]: Perhaps better "the html of the first"?

File: page/using-jquery-core/working-with-selections.md [hunk 2]
```diff
@@ -5,16 +5,17 @@
...
+$( "h1" ).html(); // "hello world"
```
Comment (AurelioDeRosa) [inline page/using-jquery-core/working-with-selections.md]: I don't think our code style allows for comments to be placed on the same line of a statement.

Comment (arthurvr) [inline page/using-jquery-core/working-with-selections.md]: We had a discussion on how we should display output of specific code in examples. On my phone right now but will dig that up tomorrow.

Comment (gnarf) [inline page/using-jquery-core/working-with-selections.md]: I think for showing return values we've waffled a few times -- We have examples in multiple formats on learn... 

https://github.com/jquery/learn.jquery.com/blob/master/page/using-jquery-core/traversing.md seems to use 

```js
// Returns "hello world"
$( "h1" ).text();
```

A lot of other pages use the trailing "returns comment"

**Comment by agcolom** [comment]
I definitely like the proposed text! 
As regards to the comments on the same line, I thought that was ok for code output but I can't find it in the style guide so we'll wait for @arthurvr 's feedback on that. If the outcome is that we have agreed this is ok, then we must remember to update the http://contribute.jquery.org/style-guide/js/

**Comment by gnarf** [comment]
I think it's not actually a "js" exception, but rather a "documentation / content" exception to that rule...  We should probably list it on a "Documentation/Content Style Guide" along with the prose rules, etc.  Link to and list an exception to the JS Style Guide.

**Comment by arthurvr** [comment]
We do have an open contribute.jquery.org PR that adds a prose/documentation style guide, this will be something to include there. I'm petty sure we had this whole discussion earlier, let me search for it tomorrow.

**Comment by agcolom** [comment]
I believe this is where we had this discussion: https://github.com/jquery/contribute.jquery.org/issues/75#issuecomment-78273631

We agreed to use ```// >``` on its own line after the code, see details at https://github.com/jquery/globalize/issues/313

File: page/using-jquery-core/working-with-selections.md (hunk 1 above)
Comment (lady3bean) [inline page/using-jquery-core/working-with-selections.md]: Yup, that sounds better.

**Comment by lady3bean** [comment]
Thanks for the input. I've updated it to reflect this style.

**Comment by gnarf** [comment]
I'm totally :+1: here - waiting for someone else to confirm before landing.

**Comment by AurelioDeRosa** [comment]
LGTM :+1:

File: page/using-jquery-core/working-with-selections.md [hunk 3]
```diff
@@ -5,16 +5,18 @@
...
+// The .html() method sets all the h1 elements' html to be "hello world":
 $( "h1" ).html( "hello world" );
 ```
 
 ```
-// The .html() method used as a getter:
+// The .html() method returns the html of the first h1 element:
 $( "h1" ).html();
+// > "hello world"
+
```
Comment (arthurvr) [inline page/using-jquery-core/working-with-selections.md]: Why is this empty line here?

**Pull request closed without merging**